### ISSUE TITLE
multi_index not accepting tuples

### ISSUE THREAD

**Issue by bkmartinjr** [TileDB-Inc/TileDB-Py#262]
The [UDF apply docs](https://docs.tiledb.com/cloud/client-api/serverless-udfs#multi-index-usage) imply the tuples and slices are interchangeable.  The standard API throws an exception on tuples (for both tiledb: and s3:)

```
>>> import tiledb, tiledb.cloud
>>> A = tiledb.DenseArray("tiledb://TileDB-Inc/quickstart_dense", ctx=tiledb.cloud.Ctx())
>>> A.multi_index[[(1,2), 4], [slice(1,4)]]['a']
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ec2-user/venv/lib64/python3.7/site-packages/tiledb/multirange_indexing.py", line 121, in __getitem__
    ranges = self.getitem_ranges(idx)
  File "/home/ec2-user/venv/lib64/python3.7/site-packages/tiledb/multirange_indexing.py", line 109, in getitem_ranges
    subranges = sel_to_subranges(sel)
  File "/home/ec2-user/venv/lib64/python3.7/site-packages/tiledb/multirange_indexing.py", line 60, in sel_to_subranges
    raise TypeError("Unsupported selection ")
TypeError: Unsupported selection 
```

It would be great if multi_index and UDF apply accepted the same index types.

**Comment by ihnorton** [comment]
Thanks for the bug report @bkmartinjr! We are doing a release tomorrow so this will be available in binaries shortly.

**Comment by ihnorton** [comment]
@bkmartinjr binaries are available for TileDB-Py 0.5.6 with the fix (tested with your query above).
